inc/TRestAxionWolterOptics.h

```diff
@@ -47,6 +53,9 @@ class TRestAxionWolterOptics : public TRestAxionOptics {
     /// Mirror thickness in mm. See schematic figure.
     std::vector<Double_t> fThickness;  //!
 
+    /// Distance between mirror stacks in mm. See schematic figure.
+    std::vector<Double_t> fXSep;  //!
```

Comment:
Remind me: For the LLNL optics, was it the same value for x_sep for every layer or a different one?

Reply:
According to my memory there was no information about a set x_sep for NuSTAR optics and they were just aligned in the front and the back which should be fine with this because it just assumes lMirror to always be the same and then the difference between one x_sep to the next should be about 0.03mm which would be what you loose by tilting the mirror further (don't know if that makes sense). But I do think we should align the mirrors in the middle with this calculation.

Reply:
> Remind me: For the LLNL optics, was it the same value for x_sep for every layer or a different one?

What Jaime and me agreed on is that probably the right thing to do for LLNL is to say x_sep = 4 mm *along the optical axis*. From there you place the cones, which due to their l = 225 mm then means the physical x_sep for each layer is slightly different and larger. If one were to align the mirrors along the rear (1st set of mirrors) and front (2nd set of mirrors) one would recover x_sep = 4 mm exactly, at the expense of having ragged mirrors (like a staircase) at the entrance and exit of the optic.

What one does in the end is not going to matter /too much/ (you see the difference though if you look at the illumination of each shell in my experience). But I would document the choice you make explicitly. That's my only point.